### PR TITLE
chore: Fixes versions and changelogs

### PR DESCRIPTION
**Pull request by oblique**
## Notes & open questions

I noticed some mistakes in the versions and in changelogs, introduced by https://github.com/libp2p/rust-libp2p/pull/5347 and some other PRs.

~~I carefully checked all changelogs and added `- unreleased` tag on their latest unreleased version to help avoiding such issues in the future. That tag should be removed before releasing and added if a new unreleased version is introduced.~~

I suggest you to check [release-plz](https://release-plz.ieni.dev/).

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

**Comment by dariusc93** [comment]
I thought it was discussed to leave the `unreleased` suffix omitted from the changelog or was it decided to reintroduce it?

**Comment by oblique** [comment]
I was not aware. @jxs Do you know?

**Comment by dariusc93** [comment]
Though I do remember in a PR I was told to drop the suffix, but this is the reference I can find about it https://github.com/libp2p/rust-libp2p/discussions/4541#discussioncomment-7077326. 

EDIT:
[comment that mentioned about leaving it out](https://github.com/libp2p/rust-libp2p/pull/4568#discussion_r1626784447)

**Comment by oblique** [comment]
Thanks. I removed the `unreleased` tag.

File: transports/websocket/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 0.44.0
```
Comment (jxs) [inline transports/websocket/CHANGELOG.md]:
```suggestion
## 0.43.1
```

File: transports/websocket/Cargo.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name = "libp2p-websocket"
 edition = "2021"
 rust-version = { workspace = true }
 description = "WebSocket transport for libp2p"
-version = "0.43.0"
+version = "0.44.0"
```
Comment (jxs) [inline transports/websocket/Cargo.toml]:
```suggestion
version = "0.43.1"
```

File: Cargo.toml [hunk 3]
```diff
@@ -112,10 +112,10 @@ libp2p-upnp = { version = "0.2.2", path = "protocols/upnp" }
 libp2p-webrtc = { version = "0.7.1-alpha", path = "transports/webrtc" }
 libp2p-webrtc-utils = { version = "0.2.1", path = "misc/webrtc-utils" }
 libp2p-webrtc-websys = { version = "0.3.0-alpha", path = "transports/webrtc-websys" }
-libp2p-websocket = { version = "0.43.0", path = "transports/websocket" }
+libp2p-websocket = { version = "0.44.0", path = "transports/websocket" }
```
Comment (jxs) [inline Cargo.toml]:
```suggestion
libp2p-websocket = { version = "0.43.1", path = "transports/websocket" }
```

**Review by jxs — APPROVED** [review]
Thanks Yianis! Yeah Darius is right we deprecated the usage of `unreleased`

**Pull request merged**